tests/main/interfaces-removable-media/task.yaml

```diff
@@ -0,0 +1,69 @@
+summary: Ensure that the removable-media interface works.
+
+details: |
+    The removable-media interface allows to access to removable storage filesystems.
+
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local test-snapd-removable-media
+
+    if ! [ -d /media ]; then
+        mkdir -p /media
+        touch /media/fake
+    fi
+    mkdir -p /media/testdir
```

Comment:
Just a remark that this test is a little unrealistic in that in real life /media would show _either_ /media or /run/media, depending on how snap-confine is configured at build time. Still, this is sufficient for the moment and we can improve it once that distinction is handled automatically by snap-confine.